worlds/earthbound/Rom.py

```diff
@@ -0,0 +1,853 @@
+from .modules.psi_shuffle import write_psi
+from .game_data.text_data import barf_text, text_encoder
+from .modules.flavor_data import flavor_data, vanilla_flavor_pointers
+from .modules.hint_data import parse_hint_data
+from .modules.enemy_data import scale_enemies
+from .modules.area_scaling import calculate_scaling
+from .modules.boss_shuffle import write_bosses
+from .modules.equipamizer import randomize_armor, randomize_weapons
+from .modules.music_rando import music_randomizer
+from .modules.palette_shuffle import randomize_psi_palettes, map_palette_shuffle
+from .modules.shopsanity import write_shop_checks
+from .modules.enemy_shuffler import apply_enemy_shuffle
+from .modules.dungeon_er import write_dungeon_entrances
+# from .modules.foodamizer import randomize_food
+from .modules.enemizer.randomize_enemy_attributes import randomize_enemy_attributes
+from .modules.enemizer.randomize_enemy_stats import randomize_enemy_stats
+from .modules.enemizer.randomize_enemy_attacks import randomize_enemy_attacks
+from .game_data.static_location_data import location_groups
+from BaseClasses import ItemClassification
+from typing import TYPE_CHECKING, Optional
+from logging import warning
+# from .local_data import local_locations
+
+if TYPE_CHECKING:
+    from . import EarthBoundWorld
+
+item_id_table = local_data.item_id_table
+location_dialogue = local_data.location_dialogue
+present_locations = local_data.present_locations
+psi_locations = local_data.psi_locations
+npc_locations = local_data.npc_locations
+character_locations = local_data.character_locations
+special_name_table = local_data.special_name_table
+item_space_checks = local_data.item_space_checks
+local_present_types = local_data.local_present_types
+present_text_pointers = local_data.present_text_pointers
+psi_item_table = local_data.psi_item_table
+character_item_table = local_data.character_item_table
+party_id_nums = local_data.party_id_nums
+starting_psi_table = local_data.starting_psi_table
+badge_names = local_data.badge_names
+world_version = local_data.world_version
+protection_checks = local_data.protection_checks
+protection_text = local_data.protection_text
+nonlocal_present_types = local_data.nonlocal_present_types
+ap_text_pntrs = local_data.ap_text_pntrs
+money_item_table = local_data.money_item_table
+
+valid_hashes = ["a864b2e5c141d2dec1c4cbed75a42a85",  # Cartridge
+                "6d71ccc8e2afda15d011348291afdf4f"]  # VC
+
+
+class LocalRom(object):
+
+    def __init__(self, file: bytes, name: Optional[str] = None) -> None:
+        self.file = bytearray(file)
+        self.name = name
+
+    def read_byte(self, offset: int) -> int:
+        return self.file[offset]
+
+    def read_bytes(self, offset: int, length: int) -> bytes:
+        return self.file[offset:offset + length]
+
+    def write_byte(self, offset: int, value: int) -> None:
+        self.file[offset] = value
+
+    def write_bytes(self, offset: int, values) -> None:
```

Comment:
```suggestion
    def write_bytes(self, offset: int, values: Sequence[int]) -> None:
```
This matches `list[int]`, `bytes`, and `bytearray`.